congestion/cubic_sender.go

```diff
@@ -76,15 +79,22 @@ func NewCubicSender(clock Clock, rttStats *RTTStats, reno bool, initialCongestio
 	}
 }
 
+// TimeUntilSend returns when the next packet should be sent.
 func (c *cubicSender) TimeUntilSend(now time.Time, bytesInFlight protocol.ByteCount) time.Duration {
```

Comment:
I think it would be cleaner if this function returned an instant, not a duration. Then you wouldn't need the now param and could drop the double diff calculation here and in the session.